assets/js/modules/sign-in-with-google/components/settings/SettingsView.js

```diff
@@ -0,0 +1,216 @@
+		const text = select( MODULES_SIGN_IN_WITH_GOOGLE ).getText();
+
+		return SIGN_IN_WITH_GOOGLE_TEXTS.find( ( { value } ) => {
+			return value === text;
+		} )?.label;
+	} );
+
+	const buttonThemeLabel = useSelect( ( select ) => {
+		const theme = select( MODULES_SIGN_IN_WITH_GOOGLE ).getTheme();
+
+		return SIGN_IN_WITH_GOOGLE_THEMES.find( ( { value } ) => {
+			return value === theme;
+		} )?.label;
+	} );
+
+	const oneTapEnabled = useSelect( ( select ) =>
+		select( MODULES_SIGN_IN_WITH_GOOGLE ).getOneTapEnabled()
+	);
+
+	const generalSettingsURL = useSelect(
+		( select ) =>
+			`${ select( CORE_SITE ).getAdminURL() }options-general.php`
+	);
+
+	const anyoneCanRegisterNoticeDismissed = useSelect( ( select ) => {
+		return select( CORE_USER ).isItemDismissed(
+			'sign-in-with-google-anyone-can-register-notice'
+		);
+	} );
+
+	// If Sign in with Google does not have a client ID, do not display the
+	// settings view.
+	if ( ! clientID ) {
+		return null;
+	}
+
+	return (
+		<div className="googlesitekit-setup-module googlesitekit-setup-module--sign-in-with-google">
+			<StoreErrorNotices
+				moduleSlug="sign-in-with-google"
+				storeName={ MODULES_SIGN_IN_WITH_GOOGLE }
+			/>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'Client ID', 'google-site-kit' ) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting value={ clientID } />
+					</p>
+				</div>
+			</div>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'Button text', 'google-site-kit' ) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting value={ buttonTextLabel } />
+					</p>
+				</div>
+
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'Button theme', 'google-site-kit' ) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting value={ buttonThemeLabel } />
+					</p>
+				</div>
+
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'Button shape', 'google-site-kit' ) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting value={ buttonShapeLabel } />
+					</p>
+				</div>
+			</div>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'One tap sign in', 'google-site-kit' ) }
```

Comment:
"One Tap" should be capitalized https://developers.google.com/identity/gsi/web/guides/features 

[The design shows this as well](https://www.figma.com/design/h9yGR15Pg2DUtgERW4AFAv/Sign-in-with-Google?node-id=27-1075&m=dev)

```suggestion
						{ __( 'One Tap sign in', 'google-site-kit' ) }
```